### DISCUSSION TITLE
Muller-Lyer and Framed Line task

### DISCUSSION THREAD

**Discussion by ccmurray924** [jspsych/jsPsych#1008] [Q&A]
I'm programming the [Muller-Lyer](https://www.illusionsindex.org/ir/mueller-lyer) and [Framed-Line](http://www.labinthewild.org/studies/frameandline/) task for an experiment, and was wondering if anyone's had experience programming these particular experiments? I'm thinking about using the image slider response plugin but am not sure of the limitations on its adaptability.

Cheers!

**Comment by jodeleeuw** [comment]
I think the [reconstruction plugin](https://www.jspsych.org/plugins/jspsych-reconstruction/) is the right choice here. It will let you control an image with keys, just like labinthewild's implementation of the framed line task.
